server/handler.go

```diff
@@ -158,6 +158,106 @@ func (p *Plugin) handleActivity(activity msteams.Activity) error {
 	return nil
 }
 
+func (p *Plugin) handleUpdatedActivity(activity msteams.Activity) error {
+	activityIds := msteams.GetActivityIds(activity)
+	if activity.ClientState != generateHash(activityIds.TeamID, activityIds.ChannelID, p.configuration.WebhookSecret) {
+		p.API.LogError("Unable to process activity", "activity", activity, "error", "Invalid webhook secret")
+		return errors.New("Invalid webhook secret")
+	}
+	var msg *msteams.Message
+	if activityIds.ReplyID != "" {
+		var err error
+		msg, err = p.msteamsBotClient.GetReply(activityIds.TeamID, activityIds.ChannelID, activityIds.MessageID, activityIds.ReplyID)
+		if err != nil {
+			p.API.LogError("Unable to get original post", "error", err)
+			return err
+		}
+	} else {
+		var err error
+		msg, err = p.msteamsBotClient.GetMessage(activityIds.TeamID, activityIds.ChannelID, activityIds.MessageID)
+		if err != nil {
+			p.API.LogError("Unable to get original post", "error", err)
+			return err
+		}
+	}
+
+	if msg.UserID == "" {
+		p.API.LogDebug("Skipping not user event", "msg", msg)
```

Comment:
These log messages will eventually be taken out right?

Reply:
Well.. it is a `LogDebug`, I'm ok removing this, but I feel it is ok to keep it there for debugging purposes.